concourse/util.py

```diff
@@ -179,6 +179,23 @@ def own_running_build_url(cfg_factory=None):
     )
 
 
+def meta_info_file_from_env() -> str:
+    meta_dir = os.path.join(
+        os.path.abspath(check_env('CC_ROOT_DIR')),
+        concourse.model.traits.meta.DIR_NAME
+    )
+    return os.path.join(
+        meta_dir,
```

Comment:
os.path.join accepts an arbitrary amount of arguments (so you might combine those two calls)

also, rather invoke os.path.abspath for the resulting path, rather just the root-dir)